packages.json

```diff
@@ -10,8 +10,8 @@
   "https://github.com/123flo321/pogoprotos-swift.git",
   "https://github.com/1904labs/ios-test-utils.git",
   "https://github.com/1fr3dg/ResourcePackage.git",
-  "https://github.com/1fr3dg/SimpleEncrypter.git",
-  "https://github.com/1fr3dg/TextFormater.git",
+  "https://github.com/1Fr3dG/SimpleEncrypter.git",
+  "https://github.com/1Fr3dG/TextFormater.git",
```

Comment:
Need to look at these changes in more detail. We should try and avoid churning on case-insensitive but it'd be good to understand which spelling is from which source.

Reply:
Fixed in 175a714a90894d49f0483a3a02f79e40e0af99c7

Reply:
So is this change good, or should we revert this one?

Reply:
This won't be in the diff when the next run completes (in 1-1.5h). We can just ditch this PR once the new one is up. Hopefully all three issues in here will be resolved.

Reply:
Ah, nice. I didn't spot that it was re-running.

Reply:
Sweet, this has now completed and updated the PR in place. We're down to just 7 legitimate additions 👌 

I'll mark this PR here as ready to merge and prep the actual nightly update one for merging as well. We can deal with the 24h breakdown separately.